berza/src/main/java/rs/edu/raf/banka/berza/bootstrap/BootstrapData.java

```diff
@@ -68,5 +73,33 @@ public void run(String... args) throws Exception {
             valute.add(v);
         }
         valutaRepository.saveAll(valute);
+
+        //Dodavanje informacija o berzi
+        List<Berza> berze = new ArrayList<>();
+        fileName = "resources/berze.csv";
+
+        List<BerzaCSV> berzeCSV = new CsvToBeanBuilder(new FileReader(fileName))
+                .withType(CurrencyCSV.class)
+                .withSkipLines(1)
+                .build()
+                .parse();
+
+        for(BerzaCSV bc: berzeCSV) {
+            Berza berza = new Berza();
+            berza.setDrzava(bc.getCountry());
+            berza.setNaziv(bc.getExchangeName());
+            berza.setOznakaBerze(bc.getExchangeAcronym());
+            berza.setVremenskaZona(bc.getTimeZone());
+            berza.setMicCode(bc.getExchangeMicCode());
+            berza.setOpenTime(bc.getOpenTime());
+            berza.setCloseTime(bc.getCloseTime());
+            //Samo dodat string valuta mora da se napravi nova, tj. da se stavi neka metoda u repozitorijum
+            //po kojoj cemo da setujemo valutu npr. valutaRepo.findByCountry(bc.getCountry()) ALI treba prvo podesiti
+            //da nazivi iz drzava iz valute.csv odgovaraju onima iz berze.csv ..
```

Comment:
Problem je što u valutama nemamo country, već samo oznaku i naziv valute. Možda možemo da probamo da matchujemo naziv valute sa onim što je u CSV fajlu, ali nemamo garanciju da će to da uspe.

Reply:
Mogu ja da namestim sve to ali tek posle srede, nadam se da nije toliko bitno za ovaj rok do utorka uvece..